crates/searchers/openapi.yaml

```diff
@@ -0,0 +1,584 @@
+        - $ref: "#/components/schemas/StablePool"
+        - $ref: "#/components/schemas/ConcentratedLiquidityPool"
+        - $ref: "#/components/schemas/ForeignLimitOrder"
+
+    Liquidity:
+      description: |
+        On-chain liquidity that can be used in a solution. This liquidity is
+        provided to facilitate TODO
+      allOf:
+        - $ref: "#/components/schemas/LiquidityParameters"
+        - type: object
+          properties:
+            address:
+              $ref: "#/components/schemas/Address"
+            gasEstimate:
+              type: integer
+              description: |
+                A rough approximation in the number gas units required to
+                use this liquidity on-chain.
+
+    Auction:
+      description: |
+        The abstract auction instance in order to solve.
+      type: object
+      properties:
+        id:
+          type: integer
+          description: |
+            The identifier for the auction. Will be set to `null` for requests
+            that are not part of an auction (when quoting token prices for
+            example).
+        tokens:
+          type: object
+          additionalProperties:
+            $ref: "#/components/schemas/Token"
+          description: |
+            A map of token addresses to token information.
+        orders:
+          type: array
+          items:
+            $ref: "#/components/schemas/Order"
+          description: |
+            The solvable orders included in the auction.
+        liquidity:
+          type: array
+          items:
+            $ref: "#/components/schemas/Liquidity"
+          description: |
+            On-chain liquidity that can be used by the solution.
+        gasPrice:
+          $ref: "#/components/schemas/TokenAmount"
+          description: |
+            The current estimated gas price. This is the price that will be used
+            for solution scoring.
+
+    JitOrder:
+      description: |
+        A just-in-time liquidity order included in a settlement.
+      type: object
+      properties:
+        sellToken:
+          $ref: "#/components/schemas/Token"
+        buyToken:
+          $ref: "#/components/schemas/Token"
+        receiver:
+          $ref: "#/components/schemas/Address"
+        sellAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        buyAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        validTo:
+          description: |
+            Unix timestamp until the order is valid to.
+          type: integer
+        appData:
+          $ref: "#/components/schemas/AppData"
+        feeAmount:
+          $ref: "#/components/schemas/TokenAmount"
+        kind:
+          $ref: "#/components/schemas/OrderSide"
+        partiallyFillable:
+          type: boolean
+        sellTokenBalance:
+          $ref: "#/components/schemas/SellTokenSource"
+        buyTokenBalance:
+          $ref: "#/components/schemas/BuyTokenDestination"
```

Comment:
For these fields (rather, all fields in `JitOrder`), I kept them the exact field names from the smart contract used for signing:

https://github.com/cowprotocol/contracts/blob/d043b0bfac7a09463c74dfe1613d0612744ed91c/src/contracts/libraries/GPv2Order.sol#L26-L48

This is because, conceptually, a `JitOrder` is a "plain" protocol order that the searcher wants to include, and I thought it made sense for it to be a 1:1 representation of what is being signed. It felt weird to to me to have the data in the `JitOrder` not match exactly what the signature represents.